fetch.bs

```diff
@@ -3101,6 +3104,10 @@ the request.
    <a for=request>origin</a> to  <var>request</var>'s
    <a for=request>client</a>'s <a for="environment settings object">origin</a>.
 
+   <li><p>If a <var>request</var>'s <a for=request>speculative flag</a> is set, then the user agent
+   may <a for=fetch lt=terminated>terminate</a> the <a>fetch</a> and return an <a>aborted network
+   error</a>, in cases where it's reasonable to believe it will result in better user experience.
+
```

Comment:
@annevk I wasn't sure what to write here, so started by writing something. It feels a bit fuzzy to be part of the processing model, so advice on making it better would be appreciated

Reply:
How concrete is it in the implementation?

Reply:
Currently I'm not aware of implementations that do that at the moment. When discussing https://github.com/whatwg/html/pull/4115 @youennf wanted to give UAs the concrete possibility to do so, while staying conformant.